.travis.yml

```diff
@@ -18,12 +18,17 @@ before_install:
  # Show the current setuptools version
  - python -c "import setuptools; print(setuptools.__version__)"
 install:
- - pip install wheel codecov coverage
+ - pip install wheel codecov
+ # Coverage doesn't work with Python 3.2 or PyPy
+ - test "$TRAVIS_PYTHON_VERSION" = "3.2" -o "`echo $TRAVIS_PYTHON_VERSION | head -c 4`" = "pypy" || pip install coverage_pth
```

Comment:
I think we can drop python 3.2 support, given that it's not supported by tox itself.

Reply:
I'm all for dropping Python 3.2 support, because it's a pain in the butt, provided that we aren't causing any issues ourselves. If tox already doesn't _work_ under Python 3.2, then I'm for it. Unfortunately despite them no longer having the 3.2 trove classifier, the docs still say it supports it, and the testing that we do also indicates that is still the case. All that to say that I don't think we should drop support yet, because I don't want it to be me that breaks somebody's build.

Reply:
> I don't think we should drop support yet, because I don't want it to be me that breaks somebody's build.

As far as I can tell, tox hasn't supported 3.2 in a long time. 
- Removed from the trove classifiers
- Not included in CI/tox configs
- Apparently, "tox has a commitment to support the same python versions as pip", and pip doesn't support python 3.2 either.

Either way, it sounds like it might be a moot point anyway 😄.